include/tvm/runtime/crt/platform.h

```diff
@@ -39,6 +41,21 @@ extern "C" {
  */
 void __attribute__((noreturn)) TVMPlatformAbort(tvm_crt_error_t code);
 
+/*! \brief Called by the microTVM RPC server to implement TVMLogf.
```

Comment:
Is this change relevant to the PR/RFC? If not suggest break it out to a different PR.

Reply:
broken out and submitted separately